src/main/java/net/fabricmc/loom/configuration/RemappedConfigurationEntry.java

```diff
@@ -61,7 +64,7 @@ public String getRemappedConfiguration() {
 
 	public String getTargetConfiguration(ConfigurationContainer container) {
 		if (container.findByName(targetConfiguration) == null) {
-			return "compile";
+			return GradleSupport.IS_GRADLE_7_OR_NEWER ? JavaPlugin.IMPLEMENTATION_CONFIGURATION_NAME : "compile";
```

Comment:
Here's a new one for the `Constants` file on the other end of the tenary operator

Reply:
This code here is a little weird. I dont really like it overall. What it does is if the java-library plugin isnt present it will use `compile` or `impl` instead of `api` when doing `modApi`.

It might not be an awful idea to fix this. At the same time?